src/test/java/api/support/fixtures/policies/PoliciesActivationFixture.java

```diff
@@ -0,0 +1,226 @@
+
+    warmUpApplyEndpoint();
+  }
+
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * inactiveNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void setFallbackPolicies(LoanPolicyBuilder loanPolicyBuilder) {
+    use(rollingDefaultPolicies()
+      .loanPolicy(loanPoliciesFixture.create(loanPolicyBuilder))
+      .noticePolicy(noticePoliciesFixture.inactiveNotice()));
+  }
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * activeNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void use(LoanPolicyBuilder loanPolicyBuilder) {
+    use(rollingDefaultPolicies()
+      .loanPolicy(loanPoliciesFixture.create(loanPolicyBuilder)));
+  }
+
+  public void use(PoliciesActivation.PoliciesActivationBuilder builder) {
+    final PoliciesActivation policies = builder.build();
+
+    useFallbackPolicies(policies.getLoanPolicy().getId(),
+      policies.getRequestPolicy().getId(),
+      policies.getNoticePolicy().getId(),
+      policies.getOverduePolicy().getId(),
+      policies.getLostItemPolicy().getId());
+  }
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * activeNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder and noticePolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void use(LoanPolicyBuilder loanPolicyBuilder,
+    NoticePolicyBuilder noticePolicyBuilder) {
+
+    use(rollingDefaultPolicies()
+      .loanPolicy(loanPoliciesFixture.create(loanPolicyBuilder))
+      .noticePolicy(noticePoliciesFixture.create(noticePolicyBuilder)));
+  }
+
+  /**
+   * This method uses notice policy, canCirculateRolling loan policy,
+   * allowAllRequestPolicy request policy,
+   * facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   *
+   * @param noticePolicy - notice policy.
+   */
+  public void use(NoticePolicyBuilder noticePolicy) {
+    use(rollingDefaultPolicies()
+      .noticePolicy(noticePoliciesFixture.create(noticePolicy)));
+  }
+
+  public void useLostItemPolicy(UUID lostItemFeePolicyId) {
+    final PoliciesActivation policies = rollingDefaultPolicies().build();
+
+    useFallbackPolicies(policies.getLoanPolicy().getId(),
+      policies.getRequestPolicy().getId(),
+      policies.getNoticePolicy().getId(),
+      policies.getOverduePolicy().getId(),
+      lostItemFeePolicyId);
+  }
+
+  /**
+   * This method uses the loan policy, allowAllRequestPolicy request policy,
+   * activeNotice notice policy, facultyStandard overdue fine policy from
+   * the loanPolicyBuilder.
+   * @param loanPolicyBuilder - loan policy builder.
+   */
+  public void useWithActiveNotice(LoanPolicyBuilder loanPolicyBuilder) {
```

Comment:
What about this method is related to notices?

Reply:
It uses the specified loan policy + uses notice policy that enables notices. (I just copy-pasted the methods from `APITests` to this fixture class, so I'm not aware about context where this method is used).

Reply:
Hmm, ok, let's leave it alone for this work.